docs/dev-setup.md

```diff
@@ -88,10 +94,13 @@ To log in as one of the base data users, when prompted for a username and passwo
         INFO  [2017-02-10 16:44:59,902] org.eclipse.jetty.server.Server: Started @4098ms
         > Building 75% > :run
         ```
+       
+#### Login
+       
 1. Go to [http://localhost:8080/](http://localhost:8080/) in your browser.
     1. When prompted for credentials:
         - **Username:** `erin`
-        - **Password:** Leave it blank
+        - **Password:** `erin`
```

Comment:
For an existing user, password should be blank.
To simulate a new user, we use username=password